scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -623,8 +623,25 @@ public void statusChanged(SchedulerStatus status) {
                 LOGGER.warning("Unexpected scheduler status");
                 break;
         }
+
         // Update the scheduler status label
-        schedulerStatusLabel.setContents("Status:" + status.name());
+
+        String neededNodes = "0";
+        String pendingEligibleTasks = "0";
+        if (controller.getModel().getSchedulerStatistics() != null) {
+            neededNodes = controller.getModel().getSchedulerStatistics().get("NeededNodes");
```

Comment:
Maybe small optimization: only 1 call controller.getModel().getSchedulerStatistics(), if this call is expensive.